src/IotHub/IotHub/ChangeLog.md

```diff
@@ -18,22 +18,23 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
-* Added support to manage devices in an Iot Hub. New Cmdlets are:
-	- `Add-AzIotHubDevice`
-	- `Get-AzIotHubDevice`
-	- `Remove-AzIotHubDevice`
-	- `Set-AzIotHubDevice`
-* Added support to manage modules on a target Iot device in an Iot Hub. New Cmdlets are:
-	- `Add-AzIotHubModule`
-	- `Get-AzIotHubModule`
-	- `Remove-AzIotHubModule`
-	- `Set-AzIotHubModule`
-* Added cmdlet to get the connection string of a target IoT device in an Iot Hub.
-* Added cmdlet to get the connection string of a module on a target IoT device in an Iot Hub.
-* Added support to get/set parent device of an IoT device. New Cmdlets are:
-    - `Get-AzIotHubDeviceParent`
-    - `Set-AzIotHubDeviceParent`
-* Added support to manage device parent-child relationship.
+* Add support to manage modules on a target Iot device in an Iot Hub. New Cmdlets are:
+	- Add-AzIotHubModule
+	- Get-AzIotHubModule
+	- Remove-AzIotHubModule
+	- Set-AzIotHubModule
+* Add cmdlet to get the connection string of a target IoT device in an Iot Hub.
+* Add cmdlet to get the connection string of a module on a target IoT device in an Iot Hub.
+* Add support to get/set parent device of an IoT device. New Cmdlets are:
+    - Get-AzIotHubDeviceParent
+    - Set-AzIotHubDeviceParent
+* Add support to manage device parent-child relationship. New Cmdlets are:
+    - Add-AzIotHubDeviceChildren
+    - Get-AzIotHubDeviceChildren
+    - Remove-AzIotHubDeviceChildren
+* Add support to manage distributed settings per-device. New Cmdlets are:
```

Comment:
Please do not change the existing release notes, just append your changes. 
Thats because the above ones are targeting S166 relases, whose release process has begun on another branch, so changing them on master will not make any difference, and will cause conflicts later.
If you think they still need to be updated, please contact Dingmeng Xue, otherwise just restore them. Thanks.

Reply:
reverted